snoop/data/digests.py

```diff
@@ -382,26 +392,31 @@ def child_dir_to_dict(directory):
     return {
         'id': directory_id(directory),
         'content_type': 'application/x-directory',
+        'filetype': 'folder',
         'filename': directory.name,
     }
 
 
-def get_directory_children(directory):
-    limit = settings.SNOOP_DOCUMENT_CHILD_QUERY_LIMIT + 1
-    child_directory_queryset = directory.child_directory_set.order_by('name_bytes')[:limit]
-    child_directory_queryset = child_directory_queryset[:limit]
+def get_directory_children(directory, page_index=1):
+    limit = settings.SNOOP_DOCUMENT_CHILD_QUERY_LIMIT
+    child_directory_queryset = directory.child_directory_set.order_by('name_bytes')
     child_file_queryset = directory.child_file_set.order_by('name_bytes')
-    child_file_queryset = child_file_queryset[:limit]
-    incomplete = len(child_directory_queryset) == limit or \
-        len(child_file_queryset) == limit
+    print(child_directory_queryset)
```

Comment:
please remove these print outs

generally i recommend adding `import pdb; pdb.set_trace()` (maybe through a single hotkey in your editor) and exploring the objects, instead of printing the <QuerySet ...> stuff around the code